src/ucp/wireup/wireup_cm.c

```diff
@@ -226,14 +305,20 @@ ucs_status_t ucp_ep_client_cm_connect_start(ucp_ep_h ucp_ep,
     cm_lane_params.sockaddr_pack_cb           = ucp_cm_client_priv_pack_cb;
     cm_lane_params.sockaddr_connect_cb.client = ucp_cm_client_connect_cb;
     cm_lane_params.disconnect_cb              = ucp_cm_disconnect_cb;
-
     ucs_assert_always(ucp_worker_num_cm_cmpts(worker) == 1);
-    cm_lane_params.cm = worker->cms[0].cm;
-    status = uct_ep_create(&cm_lane_params, &wireup_ep->sockaddr_ep);
-    if (status == UCS_OK) {
-        ucp_ep->flags |= UCP_EP_FLAG_LOCAL_CONNECTED;
+    cm_lane_params.cm                         = worker->cms[0].cm;
+
+    status = uct_ep_create(&cm_lane_params, &cm_ep);
+    if (status != UCS_OK) {
+        /* coverity[leaked_storage] */
+        return status;
     }
-    return status;
+
+    ucp_wireup_assign_lane(ucp_ep, ucp_ep_get_cm_lane(ucp_ep), cm_ep, 0,
```

Comment:
IMO this better than use `wireup_ep->sockaddr_ep`, we can remove it after iface path is deprecated and removed.
When wireup is done CM lane (next_ep) becomes permanent lane of UCP EP
PLS note, this is the only lane available for pending some amount of time, so we have to add logic with rescheduling pending requests to TL lanes at EP reconfiguration phase

Reply:
https://github.com/openucx/ucx/pull/4445/files#r347265123